examples/credit-risk-end-to-end/04_Credit_Risk_Model_Serving.ipynb

```diff
@@ -0,0 +1,960 @@
+       "      <td>1.0</td>\n",
+       "      <td>1.0</td>\n",
+       "      <td>2.0</td>\n",
+       "      <td>35.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>13</th>\n",
+       "      <td>8</td>\n",
+       "      <td>3059.0</td>\n",
+       "      <td>2.0</td>\n",
+       "      <td>12.0</td>\n",
+       "      <td>1.0</td>\n",
+       "      <td>1.0</td>\n",
+       "      <td>4.0</td>\n",
+       "      <td>61.0</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "     ID  credit_amount  installment_commitment  duration  num_dependents  \\\n",
+       "0    18        12579.0                     4.0      24.0             1.0   \n",
+       "1   764         2463.0                     4.0      24.0             1.0   \n",
+       "2   504         1207.0                     4.0      24.0             1.0   \n",
+       "3   454         4817.0                     2.0      24.0             1.0   \n",
+       "4   453         2670.0                     4.0      24.0             1.0   \n",
+       "5     0         1169.0                     4.0       6.0             1.0   \n",
+       "6     1         5951.0                     2.0      48.0             1.0   \n",
+       "7     2         2096.0                     2.0      12.0             2.0   \n",
+       "8     3         7882.0                     2.0      42.0             2.0   \n",
+       "9     4         4870.0                     3.0      24.0             2.0   \n",
+       "10    5         9055.0                     2.0      36.0             2.0   \n",
+       "11    6         2835.0                     3.0      24.0             1.0   \n",
+       "12    7         6948.0                     2.0      36.0             1.0   \n",
+       "13    8         3059.0                     2.0      12.0             1.0   \n",
+       "\n",
+       "    existing_credits  residence_since   age  \n",
+       "0                1.0              2.0  44.0  \n",
+       "1                2.0              3.0  27.0  \n",
+       "2                1.0              4.0  24.0  \n",
+       "3                1.0              3.0  31.0  \n",
+       "4                1.0              4.0  35.0  \n",
+       "5                2.0              4.0  67.0  \n",
+       "6                1.0              2.0  22.0  \n",
+       "7                1.0              3.0  49.0  \n",
+       "8                1.0              4.0  45.0  \n",
+       "9                2.0              4.0  53.0  \n",
+       "10               1.0              4.0  35.0  \n",
+       "11               1.0              4.0  53.0  \n",
+       "12               1.0              2.0  35.0  \n",
+       "13               1.0              4.0  61.0  "
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Transform JSON into dataframe\n",
```

Comment:
can't you use the same in notebook 03 to display the response of get-online request? it's definitely much more readable as a DF

Reply:
I did two things. I kept the `request.post` query, but shortened the showing of the output (first few lines, just to show this route). I also added the SDK `get_online_features` call, to show getting features that way (and as you mentioned, the nicer output for notebook display).

I felt it good to still mention the `request.post` route because AppDevs will prefer this in some use cases. From an app perspective, it can make sense to call the features REST API endpoint, then the model REST API endpoint, all without needing to import data science libraries (like sklearn and feast).